ecommerce/extensions/checkout/views.py

```diff
@@ -194,8 +194,21 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
             'has_enrollment_code_product': has_enrollment_code_product,
             'disable_back_button': self.request.GET.get('disable_back_button', 0),
         })
+        context.update({
+            'ar_name': self.get_extended_field(self.request.user.account_details(self.request), 'arabic_name')
```

Comment:
@alfredchavez, (CC @giovannicimolin), I don't think it is a good idea to add this to our common branch since it is specific to one of our customers.

Reply:
I've managed to add a tag that gets the arabic name and displays it in a custom tag, but I had to update the packages versions in the requirements(django and ipware, as koa.3 ecommerce uses django 2.20) of https://github.com/open-craft/ecommerce-paytabs, not sure how that will impact other clients, let me know if that is ok, I am going to open a PR in that repo
Or perhaps is it better to add a custom template tag directly to ecommerce?

Reply:
@lgp171188 and I cannot push a new branch to that repo, I need permissions there 
`ERROR: Permission to open-craft/ecommerce-paytabs.git denied to alfredchavez`

Reply:
@alfredchavez The correct repo is on GitLab: https://gitlab.com/opencraft/client/kkux/ecommerce-paytabs.
Use that one instead.

Reply:
@alfredchavez I've given you access to it.

Reply:
Thanks @giovannicimolin :)

Reply:
@alfredchavez, @giovannicimolin, this will return `None` for the `arabic_name` extended profile field unless it is configured in the `extended_profile_field` `SiteConfiguration` parameter. I don't see this set up on the production instance.

Reply:
@lgp171188 would be better to return an empty string instead of `None`?(easy to setup as default)  CC @giovannicimolin